src/trio/_subprocess.py

```diff
@@ -726,7 +726,7 @@ async def read_output(
     ) -> None:
         async with stream:
             async for chunk in stream:
-                chunks.append(chunk)
+                chunks.append(chunk)  # noqa: PERF401
```

Comment:
If we care about the noqa this could probably be `chunks.extend([chunk async for chunk in stream])` though if anything that's *slower* (I think?).

Maybe we should report this as a bug to ruff.